webpack/components/WithOrganization/withOrganization.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-indent */
```

Comment:
Do we need this even after ignoring all the JSX* nodes in the lint rule?

Reply:
Yes. In this one case it is needed. I have no idea why.

Reply:
First question (longer answer), there are probably a hundred "ignoredNodes" options for the indent lint rule. I'm not sure which one was being applied in this file. It was behaving differently than the rest. Unsure as to why. 

Second question: 
 eslint --fix